### ISSUE TITLE
fetch webapp endpoints on init

### ISSUE THREAD

**Issue by j6k4m8** [j6k4m8/remarkapy#7]
https://github.com/j6k4m8/remarkapy/pull/4#discussion_r1775263881_

@simonepsp did you intent by this for us to be fetching from the reMarkable API? that would be very cool if so. not sure if/where that endpt exists?

**Comment by simonepsp** [comment]
Yes, you can get all the Remarkable endpoints from https://eu.tectonic.remarkable.com/discovery/v1/endpoints with a GET request.

These are the current ones:

```
{
    "notifications" : "eu.tectonic.remarkable.com",
    "webapp" : "webapp-prod.cloud.remarkable.engineering",
    "mqttbroker" : "vernemq-prod.cloud.remarkable.engineering"
}
```

**Comment by j6k4m8** [comment]
This is so awesome, totally going to make things WAY easier to keep up with!!
